public/app/features/datasources/__mocks__/store.navIndex.mock.ts

```diff
@@ -0,0 +1,2633 @@
+import { NavSection } from '@grafana/data';
+
+export default {
```

Comment:
```suggestion
export navIndex: NavIndex = {
```
will keep us honest if the type changes/evolves

Reply:
That's a very good point, thanks 👍